src/Logging/Logging.Debug/Extensions/LogServiceFactory.Debug.Extensions.cs

```diff
@@ -0,0 +1,20 @@
+﻿using CodeMonkeys.Logging.Debug;
+
+namespace CodeMonkeys.Logging
+{
+    public static partial class LogServiceFactoryExtensions
+    {
+        /// <summary>
+        /// Method for adding the <see cref="ServiceProvider"/> to the <see cref="LogService"/>.
+        /// </summary>
+        public static void AddDebug(this ILogServiceFactory @this)
+        {
+            Argument.NotNull(
+                @this,
+                nameof(@this));
+
+            var provider = new ServiceProvider();
+            @this.AddProvider(provider);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Refactor this method to add validation of parameter 'this' before using it.](https://app.codacy.com/gh/UltimateCodeMonkeys/CodeMonkeysMVVM/pullRequest?prid=5626772)